content/_index.md

```diff
@@ -17,106 +17,143 @@ g3Teaser:
-  f4:
-    title: Customizable Options for Data Queries
-    detail: Gen3 makes data exploration easy for users that prefer a graphical interface by providing a built-in faceted search tool for building patient cohorts or selecting subsets of data files for analysis. Alternatively, users can build and send their own custom queries to pinpoint data their interested in.
-  f5:
-    title: Security & Compliance
-    detail: Gen3 can be deployed with various levels of security and compliance and was designed to make security easy by including tools for supporting a FISMA moderate environment. Gen3 supports optional tiered access, enabling users to see only summary information for data they don't have access to.
-  f6:
-    title: Control Data Access—Or Not
-    detail: You can leave your data open to the Internet or control access at deeper levels within your own data use ontology, from the core data to the data objects. Utilizing a flexible policy engine, Gen3 lets you easily create data and service access policies and assign them to user groups.
+
+g3Company: In good company...
+
+g3Commons:
+  c1:
+    title: ACCOuNT Data Commons
+    fig: /figs/commons/acct.png
+    link: https://acct.bionimbus.org
+  c2:
+    title: AnVIL
+    fig: /figs/commons/anvil.png
+    link: https://gen3.theanvil.io
+  c3:
+    title: BloodPAC Data Commons
+    fig: /figs/commons/bloodpac.png
+    link: https://data.bloodpac.org
+  c4:
+    title: Canine Data Commons
+    fig: /figs/commons/canine.png
+    link: https://caninedc.org
+  c5:
+    title: Chicagoland COVID-19 Commons
+    fig: /figs/commons/covid19.png
+    link: https://chicagoland.pandemicresponsecommons.org
+  c6:
+    title: NCI-CRDC
+    fig: /figs/commons/crdc.png
+    link: https://nci-crdc.datacommons.io
+  c7:
+    title: OCC Environmental Data Commons
+    fig: /figs/commons/edc.png
+    link: https://portal.occ-data.org
+  c8:
+    title: GenoMEL Data Commons
+    fig: /figs/commons/genomel.png
+    link: https://genomel.bionimbus.org
+  c9:
+    title: NIDDK IBD Genetics Consortium Data Commons
+    fig: /figs/commons/ibdgc.png
+    link: https://ibdgc.datacommons.io
+  c10:
+    title: Kids-First
+    fig: /figs/commons/kf.png
+    link: https://data.kidsfirstdrc.org
+  c11:
+    title: The Medical Imaging and Data Resource Center Commons
+    fig: /figs/commons/midrc.png
+    link: https://data.midrc.org
+  c12:
+    title: AccessClinicalTrails@NIAID
+    fig: /figs/commons/nct.png
+    link: https://accessclinicaldata.niaid.nih.gov
+  c13:
+    title: BioData CATALYST
+    fig: /figs/commons/bdcat.png
+    link: https://gen3.biodatacatalyst.nhlbi.nih.gov
+  c14:
+    title: Veterans Precision Oncology Data Commons
+    fig: /figs/commons/vpodc.png
+    link: https://vpodc.org
+
```

Comment:
minor: i'm not a big fan adding all these blank lines here but no strong feelings

Reply:
I actually like blank lines when inserting new blocks but then I forget to remove them in the end.